### PR TITLE
Settings improvements: resuable EditCoordinatesField component

### PR DESCRIPTION
**Pull request by jacob-8**
#### Relevant Issues
#112, #104, #105

#### Summarize what changed in this PR (for developers)
Made reusable component for editing coordinates

#### How can the changes be tested? Please also provide applicable links using preview deployments once they are available (will require editing this message once the preview deployment is ready).
http://localhost:3041/achi/settings
http://localhost:3041/create-dictionary
http://localhost:3041/admin/dictionaries

https://living-dictionaries-git-settings-improvements-2-livingtongues.vercel.app/achi/settings
https://living-dictionaries-git-settings-improvements-2-livingtongues.vercel.app/create-dictionary
https://living-dictionaries-git-settings-improvements-2-livingtongues.vercel.app/admin/dictionaries
https://ld-parts-git-settings-improvements-2-livingtongues.vercel.app/2-settings/map

<a href="https://gitpod.io/#https://github.com/livingtongues/living-dictionaries/pull/148"></a>

### CODE REVIEW

**Comment by Danble** [comment]
@jacob-8  Wouldn't be better if in `packages\site\src\sveltefirets\index.ts` we change line 33. Instead of import `./loader` we impport from `@ld/parts/src/lib/maps/loader`? Then we can delete the duplicate file loader from site.

**Review by Danble — CHANGES_REQUESTED** [review]
@jacob-8  Wouldn't be better if in `packages\site\src\sveltefirets\index.ts` we change line 33. Instead of import `./loader` we import from `@ld/parts/src/lib/maps/loader`? Then we can delete the duplicate file loader from site.
I tested and it works! But we also need to change line 12 of `packages\site\src\sveltefirets\components\FirebaseUiAuth.svelte`

**Review by Danble — COMMENTED** [review]
I also spotted a very weird bug that seems to happen when I interact with the coordinates map (If I save or remove the coordinates) when I go back to the home page, this happens:
image
It is happening both in my local environment and in the vercel link, but it triggers more easily in my local env, I have no idea why.

File: packages/site/src/routes/create-dictionary.svelte [hunk 1]
```diff
@@ -196,68 +190,58 @@
...
-          {lat}°
-          {lat < 0 ? 'S' : 'N'},
-          {lng}°
-          {lng < 0 ? 'W' : 'E'}
-        {:else}
-          <i class="fas fa-globe-americas mr-1" />
-          {$_('create.select_coordinates', { default: 'Select Coordinates' })}
-        {/if}
-      </Button>
+      <EditableCoordinatesField
```
Comment (Danble) [inline packages/site/src/routes/create-dictionary.svelte:223]: @jacob-8  It seems to me you forgot to add `{t}` here

File: packages/site/src/routes/[dictionaryId]/settings.svelte [hunk 2]
```diff
@@ -70,7 +70,18 @@
         });
       }} />
 
-    <EditCoordinates {dictionary} />
+    <div class="mt-6" />
+    <EditableCoordinatesField
```
Comment (Danble) [inline packages/site/src/routes/[dictionaryId]/settings.svelte:74]: @jacob-8 Here we also need the i18n: `t={_}`, right?

**Review by Danble — CHANGES_REQUESTED** [review]

**Comment by jacob-8** [comment]
> a very weird bug that seems to happen when I interact with the coordinates map

It was caused by my use of windi:global in Modal.svelte in svelte-pieces. Fixed in v1.0.10.

**Pull request merged**